doc/nrf/ug_matter_overview_device_types.rst

```diff
@@ -0,0 +1,210 @@
+.. _ug_matter_device_types:
+
+Matter device types
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+A Matter device type is an officially defined collection of requirements for one or more endpoints.
+Device types are intended to ensure interoperability of different device brands on the market.
+
+All device types are defined in the Device Library specification, which is available to `Connectivity Standards Alliance`_ members.
+Each device type definition is composed of the following elements:
+
+* Device type ID
+* Device type revision
+* One or more mandatory clusters, including each cluster's minimum revision
+
+The device type definition in the Device Library specification can change over time.
+The changes are tracked using the Device Type Revision value, which is initially set to ``1``.
+Changes do not modify how the device works, but only improve its functionality.
+
+A device type can require other device types for its composition, making it a *composed device type*.
+
+Device type overview
+********************
+
+The following tables list the application device types that are supported in the Matter specification.
+
+* The description for each device type is taken from the Device Library specification.
+* The state of the device specifies whether a device type can be subject to certification.
+  The `Provisional` state indicates that the device type is not yet ready for implementation.
+* The dedicated sample column provides the link to the sample in the |NCS| that implements the given device type, if available.
+
+You can add support for device types without a dedicated sample in the |NCS| by using the :ref:`matter_template_sample` and following the :ref:`ug_matter_creating_accessory` user guide, which describes how to edit clusters of a Matter application.
+
+.. _ug_matter_device_types_lighting:
+
+Lighting device types
+=====================
+
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Device type       | Description (from Device Library specification)                              | State of the device | Dedicated sample in the |NCS|         |
++===================+==============================================================================+=====================+=======================================+
+| On/Off Light      | The On/Off Light is a lighting device that is capable of being switched on   | Certifiable         | :ref:`matter_light_bulb_sample`       |
```

Comment:
The matter_light_bulb_sample is only a Dimmable Light
AFAIK, the device shall support one application device type per endpoint, our sample supports all necessary clusters to be validated as Dimmable Light

Reply:
Removed as sample for On/Off Light.